README.md

```diff
@@ -96,3 +96,17 @@ See [our other projects][community] or [hire our team][hire] to help build your
 
 [community]: https://github.com/nimblehq
 [hire]: https://nimblehq.co
+
+## FAQ
```

Comment:
should we move this FAQ to above the `## Contributing`,  or just right after the `## Usage` part 🤔